src/content/guides/asset-management.md

```diff
@@ -29,7 +29,8 @@ __dist/index.html__
 +    <title>Asset Management</title>
     </head>
     <body>
-      <script src="./bundle.js"></script>
+-     <script src="./main.js"></script>
```

Comment:
Page starts with: `If you've been following the guides from the start, `, please remove this diff and only change `./bundle.js` to `main.js`.